Documentation/AvailableOperations.md

```diff
@@ -0,0 +1,251 @@
+```
+
+After this, you should have enough data to call `setSessionData(...)` method, which will switch OAUth tokens.
+```Swift
+public func setSessionData(data sessionData: SessionData)
+```
+### Landing URL's
+#### Supported landing URLs
+We're supporting fetching landing urls for:
+- About us
+- Terms of use
+- Privacy policy
+- Registration
+- User profile
+- Explore
+- Creation
+- Forgot password
+- Upload guidelines
+- Account dashboard
+
+#### Landing URL for Creation
+For Creation landing URL, please use:
+```Swift
+public func getLandingURL(creationId creationId: String, completion: LandingURLClosure?) -> RequestHandler
+```
+#### Other Landing URLs
+For other types of landing urls, use:
+```Swift
+public func getLandingURL(type type: LandingURLType?, completion: LandingURLClosure?) -> RequestHandler
+```
+
+### Creators management
+#### Fetching currently logged in user
+To fetch currently logged in user, use `getCurrentUser(...)` method:
+```Swift
+public func getCurrentUser(completion: UserClosure?) -> RequestHandler
+```
+
+#### Fetching single user by identifier
+To fetch user by his identifier, use `getUser(...)` method:
+```Swift
+public func getUser(userId userId: String, completion: UserClosure?) -> RequestHandler
+```
+
+#### Fetching single user account data
+Account data contains more detailed information about user. To fetch it, use `getUserAccountData(...)` method:
+```Swift
+public func getUserAccountData(userId userId: String, completion: UserAccountDetailsClosure?) -> RequestHandler
+```
+You can read more about account data [here](https://partners.creatubbles.com/api/#account-details)
+
+#### Fetching Creators
+To fetch creators of a user, use a `getCreators(...)` method:
+```Swift
+public func getCreators(userId userId: String?, pagingData: PagingData?, completion: UsersClosure?) -> RequestHandler
+```
+
+You can also fetch Creators from single group:
+```Swift
+public func getCreators(groupId groupId: String, pagingData: PagingData?, completion: UsersClosure?) -> RequestHandler
+```
+
+#### Fetching Managers
+To fetch managers (parents or teachers) of a creator, use a `getManagers(...)` method:
+```Swift
+public func getManagers(userId userId: String?, pagingData: PagingData?, completion: UsersClosure?) -> RequestHandler
+```
+If you pass `nil` as a userId, managers of currently logged in user will be returned.
+
+#### Adding single Creator
+To add single Creator, use `newCreator(...)` method.
+```Swift
+public func newCreator(data creatorData: NewCreatorData,completion: UserClosure?) -> RequestHandler
+```
+
+#### Adding multiple Creators
+To add many Creators in single API call, use `createMultipleCreators(...)`.
+Note that this method will allow you to set amount of Creators and their birth year only. Usernames will be generated based on your nickname, but Creators should be able to change them after they login.
```

Comment:
'Note that this method will allow you to set amount of Creators and their birth year only'
-> 'Note that this method allows you to set the amount of Creators...'